mitodl/ocw-hugo-themes
# sanitize facets

#### Pre-Flight checklist

#### What are the relevant tickets?
closes https://github.com/mitodl/ocw-hugo-themes/issues/657

#### What's this PR do?
This pr sanitizes facets so that arbitrary facets are not pulled in from the url to search

#### How should this be manually tested?
Run `npm run start:www`
Click around search selecting various facets. Verify that everything looks normal.

Try to enter an invalid facet value in the url, for example http://localhost:3000/search/?d=NotADepartment

Verify that the invalid facet is ignored
